app/src/main/res/values/strings.xml

```diff
@@ -53,8 +53,112 @@
     <string name="shelter_dummy_address_hint">enter address here</string>
     <string name="shelter_dummy_email_hint">enter email here</string>
 
-    <!-- Strings related to login -->
+    <!-- Strings related to Pet Grid -->
+    <string name="pet_grid_picture">This is a pet</string>
+    <string name="pet_grid_title">Pet title</string>
+
+    <!-- Strings related to Settings -->
     <string name="title_activity_login">Sign in</string>
+
```

Comment:
This looks like a leftover of some conflict resolution, these **new** strings apart from the two you added are no longer in use